docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -1,118 +1,123 @@
+
 ```
 
 ## Install a specific version of a package
 
-If the version is not specified when you use the [install](../reference/cli-reference/cli-ref-install.md) command, NuGet installs the latest version of the package. You can also install a specific version of a Nuget package:
+The `install` command installs the latest version of a package unless you specify a different version. To install a specific version of a package, use the `-Version` option:
 
 ```cli
 nuget install <packageID | configFilePath> -Version <version>
 ```
 
-For example, to add version 12.0.1 of the `Newtonsoft.json` package, use this command:
+For example, to install version 12.0.1 of the `Newtonsoft.json` package, use:
 
 ```cli
 nuget install Newtonsoft.Json -Version 12.0.1
 ```
 
-For more information on the limitations and behavior of `install`, see [Install a package](#install-a-package).
-
-## Remove a package
-
-To delete one or more packages, delete the packages you want to remove from the *packages* folder.
-
-If you want to reinstall packages, use the `restore` or `install` command.
-
 ## List packages
 
-You can display a list of packages from a given source using the [list](../reference/cli-reference/cli-ref-list.md) command. Use the `-Source` option to restrict the search.
+Use the [list](../reference/cli-reference/cli-ref-list.md) command to display a list of packages. Use the `-Source` option to restrict the list.
 
 ```cli
 nuget list -Source <source>
 ```
 
-For example, list packages in the *packages* folder.
+For example, to list packages in the *packages* subdirectory of *MyProject*, use:
 
 ```cli
-nuget list -Source C:\Users\username\source\repos\MyProject\packages
+nuget list -Source C:\Users\%USERNAME%\source\repos\MyProject\packages
 ```
 
-If you use a search term, the search includes names of packages, tags, and package descriptions.
+If you don't specify a source, `list` uses all sources defined in the *NuGet.Config* global configuration file. If *NuGet.Config* doesn't specify any sources, `list` uses a default feed of nuget.org.
+
+You can also use a search term to search for package names, tags, or descriptions:
 
 ```cli
-nuget list <search term>
+nuget list <"search term"> -Source <source>
 ```
 
-## Update an individual package
-
-NuGet installs the latest version of the package when you use the `install` command unless you specify the package version.
-
 ## Update all packages
 
-Use the [update](../reference/cli-reference/cli-ref-update.md) command to update all packages. Updates all packages in a project (using `packages.config`) to their latest available versions. It is recommended to run `restore` before running `update`.
+Use the [update](../reference/cli-reference/cli-ref-update.md) command to update all packages in a project *packages.config* file to their latest available versions. It's best to run `restore` before you run `update`.
 
 ```cli
 nuget update
 ```
 
+## Remove a package
+
```

Comment:
I'd recommend that they use the PMC/PM UI. 

Just deleting a package from the packages folder does nothing for the project itself.
Notable that the packages folder tends to be in the solution file repo.

Reply:
@nkolev92 - Added some language to recommend using VS/Console.